master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -9,28 +9,77 @@ still experimental and currently comes with a number of limitations, namely:
 - Calico without etcd does not yet support Calico IPAM.  It is recommended to use `host-local` IPAM in conjunction with Kubernetes pod CIDR assignments.
 - Calico without etcd does not yet support the full set of `calicoctl` commands.
 
-## Try it out
+## Requirements 
 
 The provided manifest configures Calico to use host-local IPAM in conjunction with the Kubernetes assigned
 pod CIDRs for each node.
 
-First, ensure the following:
+You must have a cluster which meets the following requirements:
 
-- You have a Kubernetes cluster configured to use CNI network plugins (i.e by passing `--network-plugin=cni`)
-- Your Kubernetes controller manager is configured to allocate pod CIDRs (i.e by passing `--allocate-node-cidrs=true`)
-- You have configured your network to route pod traffic based on pod CIDR allocations, either through static routes or a Kubernetes cloud-provder integration.
+- You have a Kubernetes cluster configured to use CNI network plugins (i.e by passing --network-plugin=cni to the kubelet)
+- Your Kubernetes controller manager is configured to allocate pod CIDRs (i.e by passing --allocate-node-cidrs=true to the controller manager)
```

Comment:
any reason you removed the quotes - it feels like they should be there (although we do need to sort out the terrible font differences)